From bab5ea0bd7dbbf30ed13774e22b0b22ee6e3fe55 Mon Sep 17 00:00:00 2001
From: Dwane Pottratz <dwane.pottratz@intel.com>
Date: Thu, 17 Nov 2016 12:53:31 -0800
Subject: [PATCH 0028/1240] trusty: fix incompatible-pointer-types

incompatible-pointer-types found in function trusty_virtio_find_vps

drivers/trusty/trusty-virtio.c:380:14: error: initialization from
incompatible pointer type [-Werror=incompatible-pointer-types]
  .find_vqs = trusty_virtio_find_vqs,

Change-Id: Idfd949f9ca20b46537db135621bfe17ad1178d36
Signed-off-by: Dwane Pottratz <dwane.pottratz@intel.com>
---
 drivers/trusty/trusty-virtio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/trusty/trusty-virtio.c b/drivers/trusty/trusty-virtio.c
index a48f4f9..eb4c0d3 100644
--- a/drivers/trusty/trusty-virtio.c
+++ b/drivers/trusty/trusty-virtio.c
@@ -347,7 +347,7 @@ static struct virtqueue *_find_vq(struct virtio_device *vdev,
 static int trusty_virtio_find_vqs(struct virtio_device *vdev, unsigned nvqs,
 				  struct virtqueue *vqs[],
 				  vq_callback_t *callbacks[],
-				  const char *names[])
+				  const char * const names[])
 {
 	uint i;
 	int ret;
-- 
2.7.4

